Dockerfile.app

```
@@ -0,0 +1,11 @@
+FROM php:8.1.5-fpm
+
+WORKDIR /kenshu-backend-laravel
+
+COPY . .
+
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+RUN apt-get update && apt-get install -y git libzip-dev
+
+RUN docker-php-ext-install pdo_mysql
```

```suggestion
RUN docker-php-ext-install pdo_mysql zip
```
zipも入れておきましょうか、composer install時に圧縮ファイルが取り扱えなくて
```bash
  - Syncing psr/http-client (1.0.1) into cache
    Failed to download ralouphie/getallheaders from dist: The zip extension and unzip/7z commands are both missing, skipping.
Your command-line PHP is using multiple ini files. Run `php --ini` to show them.
    Now trying to download from source
```
な感じのエラーが多発してました。（環境作っただけで後で対応予定だったら申しわけないです🙏）